src/common/ad_tx.c

```diff
@@ -218,6 +268,21 @@ ad_tx_copy(struct ad_tx *tx, void *addr, daos_size_t size, void *ptr, uint32_t f
 	return 0;
 }
 
+static void
+assign_integer(void *addr, int size, uint32_t val)
+{
+	switch (size) {
+	default:
+		D_ASSERT(0);
+	case 1:
+		*((uint8_t *)addr) = (uint8_t)val;
+	case 2:
+		*((uint16_t *)addr) = (uint16_t)val;
+	case 4:
```

Comment:
(style)  Possible switch case/default not preceeded by break or fallthrough comment